src/app/components/CompanionDownloadInfo.tsx

```diff
@@ -13,10 +13,11 @@ function CompanionDownloadInfo({ hasTorCallback }: Props) {
   });
 
   function getOS() {
-    if (navigator.appVersion.indexOf("Win") != -1) return "Windows";
-    if (navigator.appVersion.indexOf("Mac") != -1) return "MacOS";
-    if (navigator.appVersion.indexOf("X11") != -1) return "UNIX";
-    if (navigator.appVersion.indexOf("Linux") != -1) return "Linux";
+    const userAgent = navigator.userAgent;
+    if (userAgent.indexOf("Win") !== -1) return "Windows";
+    if (userAgent.indexOf("Mac") !== -1) return "MacOS";
+    if (userAgent.indexOf("X11") !== -1) return "UNIX";
+    if (userAgent.indexOf("Linux") !== -1) return "Linux";
```

Comment:
n.b. this returns `undefined` when no match can be found in the userAgent string (since power users can configure the userAgent themselves, this might happen).
We then take users to this page, which fails silently (as it does for all strings at the end of the URL that are not one of the OSes listed here):
https://getalby.com/install/companion/undefined

I'd suggest the following:
1. add a fallback here (e.g. "Windows" since it's the most common OS)
2. show a warning in the Rails app when `https://getalby.com/install/companion/` is called with an unknown parameter such as `undefined` at the end.